src/parser/jobs/blu/modules/MoonFlute.tsx

```diff
@@ -72,19 +73,115 @@ export class MoonFlute extends BuffWindow {
 	override buffStatus = this.data.statuses.WAXING_NOCTURNE
 
 	private windowMissedTT = new Set<number>()
+	private breathOfMagicApplier: boolean = false
 
 	override initialise() {
 		super.initialise()
 
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
 		// Add a Waning Moon hook, essentially telling us exactly when a MF window ended.
-		const extraFilter = filter<Event>().source(this.parser.actor.id).type('statusApply')
+		const extraFilter = playerFilter.type('statusApply')
 			.status(this.data.statuses.WANING_NOCTURNE.id)
 		this.addEventHook(extraFilter, this.onWaningNocturneApply)
 
+		const bomFilter = playerFilter.type('action')
+			.action(this.data.actions.BREATH_OF_MAGIC.id)
+		this.addEventHook(bomFilter, this.onCastBreathOfMagic)
+
+		const level80BLU = this.parser.patch.after('6.4')
+		const bluLevel70MoonFlute = [
```

Comment:
This would be easier to maintain/read if you create a single `const` holding the expected actions common to both pre and post-patch BLU, then based on the patch level push additional entries into the list depending on what was expected at the time. That way we're not holding both full lists in memory regardless of patch level.

Reply:
Hm, I have the opposite take here. I think simply having the lists in plain text is much easier for a human to maintain/read, compared to having a partial list and then figuring out what gets injected into it.